docs/src/main/sphinx/connector/delta-lake.rst

```diff
@@ -722,6 +722,14 @@ directly or used in conditional statements.
 * ``$file_size``
     Size of the file for this row.
 
+.. _delta-lake-fte-support:
+
+Fault-tolerant execution support
+--------------------------------
+
+The connector supports :doc:`/admin/fault-tolerant-execution` of queries to
+Delta Lake. Read and write operations are both supported with any retry policy.
```

Comment:
with Delta Lake .. 

but thats also wrong .. hm

Maybe just leave that out and end the sentence with "of queries." or "of query processing"